documentation/navitia.io/source/integration.rst

```diff
@@ -970,6 +989,87 @@ stop_point            stop_point_               The stop point of the schedule
 Arrivals
 ********
 
+This api retrieves a list of arrivals from a datetime of a selected object.
+Arrivals are ordered chronologically in growing order.
+
+Traffic reports
+***************
+
+This service provide the state of public transport traffic.
+It can be called for an overall coverage or for a specific object. 
+
+You can access it via that kind of url: `<https://api.navitia.io/v1/{a_path_to_a_resource}/traffic_reports>`_
+
+For example:
+
+* overall public transport traffic report on Ile de France coverage
+    * https://api.navitia.io/v1/coverage/fr-idf/traffic_reports
+* Is there any perturbations on the RER network ?
+    * https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/traffic_reports
+* Is there any perturbations on the "RER A" line ?
+    * https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/lines/line:TRN:DUA810801043/traffic_reports
+
+The response is made of an array of `traffic_reports`_, and another one of `disruptions`_.
+There ares inner links between this 2 arrays: see the `inner-reference`_ section to use them.
+
+Objects
+#######
+
+.. _traffic_reports:
+
+* traffic_reports object
+
+Traffic_reports is an array of some traffic_report object. 
+One traffic_report object is a complex object, made of a network, an array of lines and an array of stop_areas.
+A typical traffic_report object will contain:
+
+* 1 network which is the grouping object
+    * it can contain links to "global network disruptions"
+* 0..n lines 
+    * each line contain at least a link to a "line disruptions"
```

Comment:
I don't find the quoted string very clear.

I would have put "each line contains at least a link to it's disruptions"
(same for the other sections)

Else it seems there are different types of disruptions